app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -45,31 +45,14 @@ class medRegistrationWorkspacePrivate
 
 medRegistrationWorkspace::medRegistrationWorkspace(QWidget *parent) : medAbstractWorkspace(parent), d(new medRegistrationWorkspacePrivate)
```

Comment:
I said though that I would prefer if all workspaces with selector toolboxes inherit from medSelectorWorkspace.